util/execdetails/execdetails.go

```diff
@@ -53,60 +53,69 @@ type CommitDetails struct {
 	TxnRetry          int
 }
 
+const (
+	ProcessTimeStr   = "Process_time"
+	WaitTimeStr      = "Wait_time"
+	BackoffTimeStr   = "Backoff_time"
+	RequestCountStr  = "Request_count"
+	TotalKeysStr     = "Total_keys"
+	ProcessedKeysStr = "Processed_keys"
+)
+
 // String implements the fmt.Stringer interface.
 func (d ExecDetails) String() string {
 	parts := make([]string, 0, 6)
 	if d.ProcessTime > 0 {
-		parts = append(parts, fmt.Sprintf("process_time:%vs", d.ProcessTime.Seconds()))
+		parts = append(parts, fmt.Sprintf("Process_time: %v", d.ProcessTime.Seconds()))
```

Comment:
I prefer the lower case format, to be convenient to `grep`.

Reply:
`pt-query-digest` won't parser this field if use `process_time`, it need capitalize.